### ISSUE TITLE
Too fast for dashboard

### ISSUE THREAD

**Issue by jano42** [Yadoms/yadoms#507]
After a full page refresh, I've opened the dashboard and relatively quickly clicked on devices tab.

The device tab start to display, but suddenly it goes back to summary.

This is very annoying.

![dashboard_slow](https://user-images.githubusercontent.com/3135683/47700997-b04d9380-dc18-11e8-9055-245cc4326a5a.gif)

**Comment by jmberhault** [comment]
Is this issue is reproducible with Yadoms 2.2.0 ? I coud not reproduce it with my configuration.

**Comment by sgallou** [comment]
Yes. More visible with a poor connection between client and server.
